lua/rustaceanvim/config/internal.lua

```diff
@@ -168,6 +168,11 @@ local RustaceanDefaultConfig = {
       --- default: false
       ---@type boolean
       auto_focus = false,
+
+      --- whether splits opened from floating preview are vertical
+      --- default: false
+      ---@type boolean
+      open_split_vertical = false,
```

Comment:
There's also a `config/init.lua`, which contains luaCATS annotations for lua-language-server, where each field can be `nil` (so that it doesn't complain about missing fields in user configs).
That would also need to be updated. 

There's a workflow that generates vimdoc from it, but for some reason it never runs when someone other than myself opens a PR 😅
I'll have to fix that. 

If you have nix installed, you could run it with `nix run .#docgen`. Otherwise, I can do it myself after merging.

Reply:
i took a look in `config/init.lua`, but it seems that there is only a top level `float_win_config`, without any of the child options in the table: https://github.com/mrcjkb/rustaceanvim/blob/2eb8776df1aab03f514b38ddc39af57efbd8970b/lua/rustaceanvim/config/init.lua#L68

i also tried running the nix command, and it seemed to run successfully, but did not update any of the docs. perhaps my nix is configured incorrectly? i don't use it much

Reply:
Oh yeah, it's a `table` because it's a built-in neovim config.
I'll merge and update the docs later.